src/Model/Post/Delayed.php

```diff
@@ -33,19 +33,23 @@
 
 class Delayed
 {
+	const PREPARED = 0;
```

Comment:
Please explain the meaning of these three values as it isn’t intuitive just from the names.

Reply:
A boolean parameter has been converted to an integer parameter. Since there can be outstanding jobs in the workerqueue during the update we have to assure that it is interpreted correctly.

Reply:
I meant in the code, please. 😅

Reply:
Done